docs/map.xsd

```diff
@@ -59,13 +131,31 @@
   </xs:restriction>
 </xs:simpleType>
 
-<xs:simpleType name="hexColourT">
+  <xs:simpleType name="cornerterrainT">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[0-9]{1,3},[0-9]{1,3},[0-9]{1,3},[0-9]{1,3}"/>
+    </xs:restriction>
+  </xs:simpleType>
+
+<xs:simpleType name="hexColorT">
   <xs:restriction base="xs:string">
-    <xs:pattern value="[a-fA-F0-9]{6}"/>
+    <xs:pattern value="([a-fA-F0-9][a-fA-F0-9]){3}"/>
   </xs:restriction>
 </xs:simpleType>
 
-<xs:complexType name="simpleDataT">
+  <xs:simpleType name="pointsT">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="(\[([0-9])+,([0-9])+\])+"/>
```

Comment:
I'm not sure if this is a good idea, because the floating point number could in theory also contain an exponent, unless you can include that in the pattern as well. In terms of documenting the format, I think something similar to what SVG did is more useful anyway.